README.md

```diff
@@ -20,14 +15,8 @@ mvn package
 Running
 -------
 
-Before running the migration utility, stop the repository by shutting down the servlet container (Tomcat, Jetty, etc.) or removing the Fedora 4 webapp.  Then run the migration utility by executing the JAR file and provide the `fcrepo.home` system property to set the location of Fedora 4's `fcrepo4-data` directory.
 
 ``` sh
-java -Dfcrepo.home=/path/to/fcrepo4-data -jar /path/to/fcrepo4-upgrade-utils/target/fcrepo-upgrade-utils-4.3.1-SNAPSHOT.jar
+java -i /path/to/4.7.x/export -o /optional/path/to/5.x/tranform-directory  -jar /path/to/fcrepo4-upgrade-utils/target/fcrepo-upgrade-utils-5.0.0-SNAPSHOT.jar
```

Comment:
I believe you should be running with java 11.

Reply:
Okay, I was running with java 8 before since that was what was in the pom file. But I get the same error in java 11 `openjdk version "11.0.6" 2020-01-14`
```
Unrecognized option: -i
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```
Are you sure the `-i` and `-o` options shouldn't be after the jar file since they seem to be the input and output parameters for the upgrade util rather than flags for java?

Reply:
They should be after the jar file. I'll update the readme.